### PR TITLE
Add OneSync, new update for FiveM which allows up to 64 slots.

### PR DESCRIPTION
**Pull request by ScottUK**

### CODE REVIEW

File: gta/fivem/egg-five-m.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     },
     "variables": [
         {
-            "name": "fivem license",
+            "name": "FiveM License",
```
Comment (parkervcp) [inline gta/fivem/egg-five-m.json]: Why upper case this?

**Comment by parkervcp** [comment]
@ScottUK Please explain what onesync is and why it should be added.

File: gta/fivem/egg-five-m.json (hunk 1 above)
Comment (ScottUK) [reply gta/fivem/egg-five-m.json]: My bad, probably just my OCD I will removei t.

**Comment by ScottUK** [comment]
OneSync allows for up to 64 slots from the normal 32 that FiveM offers. It is in Alpha and is more likely to cause crashes, however, it is pretty stable as I currently use it on my FiveM server.

**Comment by parkervcp** [comment]
I'll say no to just updating the current egg as it could cause potential issues with installs as it's an Alpha build.

Move it to a different folder and change the name to something like FiveM-OneSync

**Comment by parkervcp** [comment]
I am closing this due to lack of response.

**Pull request closed without merging**